tests/transaction_hooks/tests.py

```diff
@@ -59,6 +115,15 @@ def test_does_not_execute_if_transaction_rolled_back(self):
 
         self.assertDone([])
 
+        try:
+            with transaction.atomic():
+                self.do(1, True)
+                raise ForcedError()
```

Comment:
`()` are unnecessary
```suggestion
                raise ForcedError
```